src/renderer/components/HistoryProvider.tsx

```diff
@@ -100,7 +100,7 @@ export const HistoryProvider = ({ children }: React.PropsWithChildren<unknown>):
 
         const id = setTimeout(() => {
             historyObj.history = historyObj.history.commit([getNodes(), getEdges()]);
-        }, 500);
+        }, 250);
```

Comment:
Why?

Reply:
Because it's not needed anymore. I only increased it to 500 to try to fix the number input issues